test/unit/carousel-slide.ts

```diff
@@ -0,0 +1,50 @@
+import CarouselSlide from '../../src/carousel-slide';
+import suite from './_suite';
+
+suite('CarouselSlide', ({ expect, spy }) => {
+  let carouselSlide: CarouselSlide;
+
+  beforeEach(() => carouselSlide = new CarouselSlide());
+
+  describe('init()', () => {
+    it('should expose the slide and index', () => {
```

Comment:
This and the next test should all be 1 test, ie `'should expose the slide and index and unexpose carousel'`